src/bluetoothdevice.h

```diff
@@ -103,24 +99,22 @@ class bluetoothdevice : public QObject {
 
     /**
      * @brief jouls Gets a metric object to get and set the number of joules expended. Units: joules
-     * @return
      */
     metric jouls();
 
     /**
      * @brief fanSpeed Gets the current fan speed. Units: revolutions per second
-     * @return
```

Comment:
mmm the unit depends to the device. On domyos, for example, it's just a pure number without unit